util/settings.lua

```diff
@@ -45,6 +45,10 @@ mail.settings = {
         type = "string", default = "%Y-%m-%d %X", group = "other", index = 3, label = S("Date format"),
         dataset = {"%Y-%m-%d %X", "%d/%m/%y %X", "%A %d %B %Y %X"}, format = os.date
     },
+    timezone_offset = {
+        type = "number", default = 0, group = "other", index = 4,
+        label = S("Timezone offset"), tooltip = S("Offset to add to UTC time. DST is not supported."),
```

Comment:
It seems like the timezone offset is relative to the server time (at least this seems to be the case in singleplayer).

(DST does not need to be mentioned in this case IMO since a static offset from the server time would imply that the server-side DST settings is used).

```suggestion
        label = S("Timezone offset"), tooltip = S("Offset to add to server time."),
```